src/agent/src/mount.rs

```diff
@@ -573,6 +581,92 @@ fn mount_storage(logger: &Logger, storage: &Storage) -> Result<()> {
     )
 }
 
+#[instrument]
+pub fn set_ownership(logger: &Logger, storage: &Storage) -> Result<()> {
+    let logger = logger.new(o!("subsystem" => "change_ownership"));
```

Comment:
Here may be some miss-use of `subsystem` if log fields in this file(mount.rs). Suggest keep subsystem to `mount`  and set func to it's name by using `o!("subsystem" => "mount", "fn" => "set_ownership")`.

```rust
src/agent/src/mount.rs:150:    let logger = logger.new(o!("subsystem" => "baremount"));
src/agent/src/mount.rs:531:    let logger = logger.new(o!("subsystem" => "mount"));
src/agent/src/mount.rs:728:    let logger = logger.new(o!("subsystem" => "mount"));
src/agent/src/mount.rs:868:    let logger = logger.new(o!("subsystem" => "mount"));
```

Reply:
thanks, changed to `logger.new(o!("subsystem" => "mount", "fn" => "set_ownership"))`